lib/ruby_terraform/commands/base.rb

```diff
@@ -25,12 +26,14 @@ def execute(opts = {})
         builder = instantiate_builder
 
         do_before(opts)
-        configure_command(builder, opts)
-          .build
-          .execute(
-              stdin: stdin,
-              stdout: stdout,
-              stderr: stderr)
+        command = configure_command(builder, opts)
+                    .build
+        @logger.info "Running #{command.to_s}"
```

Comment:
Often, the command string will include sensitive information such as credentials or certificates. I think this should be a debug log rather than an info log so that consumers have to opt in to printing sensitive vars.